test/marey.js

```diff
@@ -1,160 +0,0 @@
-/**
```

Comment:
The Marey plot is also a useful debug tool and should be retained somewhere although it is not a test _per se_.